### PR TITLE
Fix old broken link

### PR DESCRIPTION
**Pull request by sthetland**
Fixing some old links. I've resolved them in various ways, either fixing the link or removing and writing the link out of the text (especially if the topic seems stale, and likely alternative link targets are hard to find).

### CODE REVIEW

File: docs/querying/aggregations.md [hunk 1]
```diff
@@ -395,7 +395,7 @@ The [Approximate Histogram](../development/extensions-core/approximate-histogram
 
 The algorithm used by this deprecated aggregator is highly distribution-dependent and its output is subject to serious distortions when the input does not fit within the algorithm's limitations.
 
-A [study published by the DataSketches team](https://datasketches.apache.org/docs/Quantiles/DruidApproxHistogramStudy.html) demonstrates some of the known failure modes of this algorithm:
+Some of the known failure modes of this algorithm include:
```
[the remark this answers is not in the document]
Comment (sthetland) [reply docs/querying/aggregations.md:398]: It seems broken to me though, and I can't find a new link for it. Are you able to still see it? If it isn't broken, nope, no reason.. let's keep it.

[the remark this answers is not in the document]
Comment (clintropolis) [reply docs/querying/aggregations.md:398]: Ah, it has changed slightly, https://datasketches.apache.org/docs/QuantilesStudies/DruidApproxHistogramStudy.html

[the remark this answers is not in the document]
Comment (sthetland) [reply docs/querying/aggregations.md]: Good find! Updated.

**Review by clintropolis — APPROVED** [review]

**Comment by suneet-s** [comment]
@sthetland do you mind merging master back in to your PR. I think there were some issues with the master branch last week.  I think CI should pass after that

**Pull request merged**